### DISCUSSION TITLE
Precommit, which adds period after sentence breaks URLS

### DISCUSSION THREAD

**Discussion by ducha-aiki** [kornia/kornia#1344] [Potential Issue] [answered]
### Describe the bug

In the title

### Reproduction steps

Example here: https://github.com/kornia/kornia/pull/1335/commits/f0b90ee1813a5051c870322a6a07a5bcd071ef85

<img width="678" alt="image" src="https://user-images.githubusercontent.com/4803565/135597468-d4ffca14-17bc-4752-83b3-c6733cafa518.png">

### Expected behavior

123

### Environment

```shell
wget https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py
# For security purposes, please check the contents of collect_env.py before running it.
python collect_env.py
```
- PyTorch Version (e.g., 1.0):
- OS (e.g., Linux):
- How you installed PyTorch (`conda`, `pip`, source):
- Build command you used (if compiling from source):
- Python version:
- CUDA/cuDNN version:
- GPU models and configuration:
- Any other relevant information:
```


### Additional context

_No response_

**Comment by edgarriba** [comment, marked as answer]
Solution should be not adding end point
